src/test/java/org/toradocu/PrecisionRecallCommonsMath3.java

```diff
@@ -22,7 +22,14 @@ public void testGaussian() throws Exception {
 
   @Test
   public void testUnivariateSolverUtils() throws Exception {
-    test("org.apache.commons.math3.analysis.solvers.UnivariateSolverUtils", 1, 0.777, 1, 0, 1, 1);
+    test(
+        "org.apache.commons.math3.analysis.solvers.UnivariateSolverUtils",
+        0.889,
```

Comment:
Yes. This is due to the following condition:
Comment:` if lower >= initial or initial >= upper`.

Here we have a composition of conditions, currently not supported by Toradocu. So, before, this resulted in a totally empty condition. Now, the first part results in a translation, but the second cannot be translated: the whole result is a wrong condition.